apps/extension/entrypoints/background.ts

```diff
@@ -37,12 +53,60 @@ export default defineBackground(() => {
         }
       })
       .catch((err) => {
-        console.error("error handling message", err)
+        console.error("error handling message", err, msg)
       })
 
     return true
   })
 
+  async function handleTlinkApiRequest(method: string, payload: any) {
+    switch (method) {
+      case "getTurnstileToken":
+      case "getRecaptchaToken":
+        return handleTlinkApiViaTSViewerWindow(method, payload);
+    }
+  }
+
+  async function handleTlinkApiViaTSViewerWindow(method: string, payload: any){
+
+    return new Promise(async (resolve, reject) => {
+
+      let popup: WindowProxy|null;
+
+      const requestUrl = `http://localhost:3333/?viewType=tlink-api&method=${method}&payload=${encodeURIComponent(JSON.stringify(payload))}`
```

Comment:
we need to change the url from local to a real prod one

Reply:
@futantan yes. Also need to change from staging to production now too